mobify/platform-scaffold
# AMP Integration Manager...integration

Sharing parsers/integration manager backend + selectors with the PWA

 **JIRA**: https://mobify.atlassian.net/browse/AMP-43, ninja-fix to https://mobify.atlassian.net/browse/AMP-64

## Changes
- ~~Bunch of shims in `main.js` to make the integration manager codepaths work~~
- ~~Bunch of pass-throughs and whatever else required in `webpack.config.js`~~
- Brought in our own ES6-ified capture.js 'disable' function
- Made `jqueryResponse` externally configurable to the merlins connector
- Implemented our own ES6-ified jqueryResponse
- Made the web scaffold a little more tolerant of running in a browserless environment
- Hooked up selectors in our existing templates

## How to test-drive this PR
- Run `npm i` from the `web` directory - this is a new requirement! (I don't have a solution to this yet)
- Run `npm run dev`
- Check out http://localhost:3000, http://localhost:3000/eye-of-newt.html, http://localhost:3000/potions.html
- Run `npm run test:all`
